services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchActivationSpec.groovy

```diff
@@ -0,0 +1,133 @@
+package org.openkilda.functionaltests.spec.switches
+
+import static org.junit.Assume.assumeTrue
+import static org.openkilda.model.MeterId.MAX_SYSTEM_RULE_METER_ID
+import static org.openkilda.model.MeterId.MIN_FLOW_METER_ID
+import static org.openkilda.testing.Constants.NON_EXISTENT_FLOW_ID
+import static org.openkilda.testing.Constants.WAIT_OFFSET
+
+import org.openkilda.functionaltests.HealthCheckSpecification
+import org.openkilda.functionaltests.helpers.SwitchHelper
+import org.openkilda.functionaltests.helpers.Wrappers
+import org.openkilda.messaging.Message
+import org.openkilda.messaging.command.CommandData
+import org.openkilda.messaging.command.CommandMessage
+import org.openkilda.messaging.command.flow.InstallEgressFlow
+import org.openkilda.messaging.command.flow.InstallIngressFlow
+import org.openkilda.messaging.command.flow.InstallTransitFlow
+import org.openkilda.messaging.command.switches.DeleteRulesAction
+import org.openkilda.model.Cookie
+import org.openkilda.model.FlowEncapsulationType
+import org.openkilda.model.OutputVlanType
+
+import org.apache.kafka.clients.producer.KafkaProducer
+import org.apache.kafka.clients.producer.ProducerRecord
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.beans.factory.annotation.Value
+
+class SwitchActivationSpec extends HealthCheckSpecification {
+    @Value("#{kafkaTopicsConfig.getSpeakerFlowTopic()}")
+    String flowTopic
+    @Autowired
+    @Qualifier("kafkaProducerProperties")
+    Properties producerProps
+    @Autowired
+    SwitchHelper switchHelper
+
+    def "Missing flow rules/meters are removed from a new switch connected to the controller"() {
```

Comment:
Missing rules are removed from the switch? Missing rules should be installed to the switch. Also, please rephrase '..new switch connected to the controller' to `..new switch upon connecting to the conntroller/when connecting to the controller/before connecting to the controller`

Reply:
fixed
`Missing flow rules/meters are installed on a new switch before connecting to the controller`